.github/actions/auto_gen_bind_pr/action.yaml

```
@@ -33,6 +33,10 @@ runs:
     - uses: actions-rs/toolchain@v1
       with:
         toolchain: stable
+    - name: Set env for iOS build
+      if: inputs.triple == 'aarch64-apple-ios' || inputs.triple == 'aarch64-apple-ios-sim' || inputs.triple == 'x86_64-apple-ios'
```

こうでも良いかもです！
```suggestion
      if: startsWith(inputs.triple, 'aarch64')
```
https://docs.github.com/ja/actions/learn-github-actions/expressions#startswith

---

なるほど、こちらでもstartsWithが使えるのですね。
このままだとx86_64のシミュレータが除外されてしまうので、containsを使って次のようにした方が良さそうです。

```yaml
if: contains(inputs.triple, 'apple-ios')
```

---

あっほんとですね！！ 🙇